internal/bminventory/inventory.go

```diff
@@ -171,7 +172,12 @@ const ignitionConfigFormat = `{
 		"name": "selinux.service",
 		"enabled": true,
 		"contents": "[Service]\nType=oneshot\nExecStartPre=checkmodule -M -m -o /root/assisted.mod /root/assisted.te\nExecStartPre=semodule_package -o /root/assisted.pp -m /root/assisted.mod\nExecStart=semodule -i /root/assisted.pp\n\n[Install]\nWantedBy=multi-user.target"
-	}{{if .StaticIPsData}},
+	},
+	{
+		"name": "pre-network-manager-config.service",
+		"enabled": true,
+		"contents": "[Unit]\nDescription=Prepare network manager config content\nBefore=NetworkManager.service\nDefaultDependencies=no\n[Service]\nUser=root\nType=oneshot\nTimeoutSec=10\nExecStart=/bin/bash /usr/local/bin/pre-network-manager-config.sh\nPrivateTmp=true\nRemainAfterExit=no\n[Install]\nWantedBy=multi-user.target"
+    }{{if .StaticIPsData}},
```

Comment:
Should this section be removed entirely and other references of StaticIPsData as part of this PR or on a separate one ?

Reply:
i am going to remove it as part of MGMT-4089